src/components/workspace-utils.js

```diff
@@ -340,6 +346,55 @@ export const WorkspaceTagSelect = props => {
   })
 }
 
+export const WorkspaceSubmissionStatusIcon = ({ status, loadingSubmissionStats, size = 20 }) => {
+  return Utils.cond(
+    [loadingSubmissionStats, () => h(DelayedRender, [
+      h(TooltipTrigger, {
+        content: 'Loading submission status',
+        side: 'left'
+      }, [spinner({ size })])
+    ])],
+    [status, () => h(TooltipTrigger, {
+      content: span(['Last submitted workflow status: ', span({ style: { fontWeight: 600 } }, [_.startCase(status)])]),
+      side: 'left'
+    }, [
+      Utils.switchCase(status,
+        ['success', () => icon('success-standard', { size, style: { color: colors.success() } })],
+        ['failure', () => icon('error-standard', { size, style: { color: colors.danger(0.85) } })],
+        ['running', () => icon('sync', { size, style: { color: colors.success() } })]
+      )
+    ])],
+    () => null)
+}
+
+export const RecentlyViewedWorkspaceCard = ({ workspace, submissionStatus, loadingSubmissionStats, timestamp }) => {
+  const { workspace: { namespace, name, googleProject } } = workspace
+
+  const dateViewed = Utils.makeCompleteDate(new Date(parseInt(timestamp)).toString())
+
+  return h(Clickable, {
+    style: { ...Style.elements.card.container, margin: '0 0.25rem 0 0.25rem', lineHeight: '22px', flex: '0 1 calc(25% - 10px)' },
```

Comment:
I think line height should be specified in `rem` so that it scales with text size.

More info on absolute vs relative units in CSS here: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#numbers_lengths_and_percentages

Reply:
margin css shorthanded in d63db7d21ea173ce492eeb165e8cb5eefcb368ec, switched lineHeight to 1.5rem in https://github.com/DataBiosphere/terra-ui/pull/3280/commits/ac62229ded3443c6ea61a12e32011d9cca11ab15